src/ubuntu/18.04/helix/arm32v7/Dockerfile

```diff
@@ -46,6 +47,13 @@ RUN apt-get update && \
     \
     && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 
+# Add MsQuic
+# MsQuic did not publish arm32 package for 18.04
+# Use direct link to signed binary for now
+RUN curl -LO https://github.com/microsoft/msquic/releases/download/v2.1.8/libmsquic_2.1.8_armhf.deb && \
+    dpkg -i libmsquic_*_arm64.deb && \
+    rm -f libmsquic_*_arm64.deb
```

Comment:
```suggestion
    dpkg -i libmsquic_*_armhf.deb && \
    rm -f libmsquic_*_armhf.deb
```
a typo?

Reply:
yes. cut & paste. I don't have ARM machine to test on so CI is the ultimate check.